doc/dev/development.rst

```diff
@@ -445,8 +491,11 @@ The tagging process is as follows:
         git add -u
         git commit -m 'tag CHANGES'
 
-        # Increment the version from 'dev' to a release version and tag
-        bumpversion release --verbose --tag --commit
+        # Create a tag of the form X.Y.Z (using 1.3.1 here as an example).
+        # The current autogenerated version is found in pypeit/version.py
+        # and the tag used here should be everything in front of the 'dev'
+        # string in the current version.
+        git tag 1.3.1
```

Comment:
that's because you're pulling off my fork which doesn't have the same tags as the main repo. i noticed this same thing. it's correct given the latest tag in my fork and i played around with tags in various checkouts to make sure `setuptools_scm` is doing the right thing and it is. so once this is merged to `master` and a tag of `1.3.1` set there, it should all be gtg.

Reply:
the back story is that my fork is quite old. i can try to update the tags there, if you want. probably want to do the 1.3 -> 1.3.0 fix first, though.

Reply:
I vote that we just merge this PR now and then hopefully you can help me fix the version numbering when it makes it into develop.